src/components/Layout/LinesParams.tsx

```diff
@@ -99,14 +99,10 @@ export const LinesParams = ({ address, method, params }: LinesParamsProps) => {
     <BaseTypeObjLines label='Params' data={decodedParams} />
   ) : isDelegated && !abi ? (
     <Line label='Params (upload abi to decode)'>{params}</Line>
-  ) : params ? (
-    <Line
-      label={`Params (${
-        decodeParamsError ? 'failed to decode' : 'decoding...'
-      })`}
-    >
-      {params}
-    </Line>
+  ) : params && decodeParamsError ? (
+    <Line label='Params (failed to decode)'>{params}</Line>
+  ) : params && !isDelegated ? (
+    <Line label='Params (decoding...)'>{params}</Line>
```

Comment:
I think this seems to prevent a `(decoding...)` state for the params when the actor is delegated, which might make sense because parameter decoding is synchronous. But I think it also causes to now just show the `NullishLine` if the parameters for a smart contract tx fail to decode for some reason.